docs/configuration-files.md

```diff
@@ -19,6 +19,7 @@ Contents:
   * [`pattern-either`](configuration-files.md#pattern-either)
   * [`pattern-regex`](configuration-files.md#pattern-regex)
   * [`metavariable-regex`](configuration-files.md#metavariable-regex)
+  * [`metavariable-comparison`](configuration-files.md#metavariable-comparison)
```

Comment:
could also be called pattern-where-python-safe

Reply:
Maybe the only safe thing people want to express are comparisons, so maybe it's not a bad name but it does not really give room for being more than that. For example some people wanted to check if $X in [1, 42, 57]
and it's not really a comparison (or maybe it is).

Reply:
sounds fair.

Reply:
I like the resolution here. I also would add that this suggestion would make the unsafe variant our de facto default, which is something we should always strive to avoid.